dcos_installer/upgrade.py

```diff
@@ -74,13 +74,17 @@
 # Check if the node has node/cluster checks and run them
 if [ -f /opt/mesosphere/etc/dcos-diagnostics-runner-config.json ]; then
    # command exists
-   if !(output=$(dcos-diagnostics check node-poststart)); then
+   output=$(dcos-diagnostics check node-poststart)
+   ret=$?
+   if [ $ret -ne 0 ]; then
```

Comment:
just curious, is there a benefit of assigning `$?` to `ret`?
instead of just comparing:
`if [ $? -ne 0 ]; then`